waiter/src/waiter/process_request.clj

```diff
@@ -555,13 +555,15 @@
                                   (process-backend-response-fn local-usage-agent instance-request-properties descriptor instance request
                                                                reason-map reservation-status-promise confirm-live-connection-with-abort
                                                                request-state-chan response))
+                                (assoc :backend-response-latency-ns response-elapsed)
```

Comment:
`response-elapsed` is created in https://github.com/twosigma/waiter/pull/248/files#diff-d77fcc9f68a157c1df598bfad7a0ccf4R546
Can an exception be thrown between then and this line? If so, this value will end up not being logged.